### ISSUE TITLE
Support for rancher 2.x?

### ISSUE THREAD

**Issue by tibers** [greensheep/terraform-aws-rancher-server#7]
Title says it all - I was googling for a way to provision rancher in AWS and I found this but it looks like it's only for rancher 1.x.

**Comment by greensheep** [comment]
That's right, it's only Rancher 1.x at the moment. To be honest I've not really looked at Rancher 2 so there's no immediate plans.

**Comment by tibers** [comment]
Not really relevant to the issue at hand but Rancher 2.x's API is really nice and fairly easy to work with once you get the hang of it. Thanks for the heads up about this repo.
